### DISCUSSION TITLE
Custom Achivements

### DISCUSSION THREAD

**Discussion by Jak2k** [community/community#30209] [Other Feature Feedback, Questions, & Ideas] [answered]
Custom achivments would be great!

**Comment by Angel-del-dev** [comment]
What do you mean by custom achievements? Like being able to set your own like goals?

**Comment by NikhilMath** [comment, marked as answer]
unfortunately they will not be doing that because of how the system is set up. But I do agree with you. Having a specific badge you made would be cool, or have ones your friends can opt in.

**Comment by antonmedv** [comment]
I also wanted custom achievements so I created it: https://github.com/my-badges/my-badges

Check this out: 
<img width="1287" alt="Screenshot 2023-10-05 at 22 20 12" src="https://github.com/community/community/assets/141232/b0ab63bd-6b67-43b2-8230-589bda624b58">

**Comment by Loyydowski** [comment]
I totally agree!
